### DISCUSSION TITLE
Lint-staged and Eslint keeps scanning my snapshots test files

### DISCUSSION THREAD

**Discussion by santanapaulo** [eslint/eslint#16783] [Q&A] [answered]
Hello everyone! I think my question is simple to answer (not for me).

Is there a way I can work with lint-staged and eslint without go through my snapshot (jest) test files?
My eslint is always complaning about some error on my snapshot.

I am attaching this images down below so you guys can check if there is something wrong.

![image](https://user-images.githubusercontent.com/15819810/212399709-370848ce-65d7-405b-ad68-948c2cdad65e.png)
![image](https://user-images.githubusercontent.com/15819810/212399549-bbeda77d-563e-4c7c-a791-8fa7de2d98f1.png)
![image](https://user-images.githubusercontent.com/15819810/212399614-ddee0d69-149e-4221-a49b-24dc66a616a4.png)

**Comment by santanapaulo** [comment, marked as answer]
LOL

After a few hours trying to figure it out what to do, just came in my mind the idea of changing the files lint-staged will go through. Not sure if this is the best approach, but seems to be worth it.

![image](https://user-images.githubusercontent.com/15819810/212401773-3f704b46-7d2d-4d7e-a85c-0d4fa64872b0.png)

**Reply by nzakas** [reply]
Yes, that's the correct approach. In general, it's a bad idea to use `**/*` with lint-staged because not all files will need to the same precommit treatment.
